modin/core/storage_formats/pandas/parsers.py

```diff
@@ -855,11 +881,14 @@ def parse(fname, **kwargs):
         if num_splits is None:
             return pandas.read_feather(fname, **kwargs)
 
+        to_pandas_kwargs = PandasParser.get_types_mapper(kwargs["dtype_backend"])
+        del kwargs["dtype_backend"]
```

Comment:
Why do we need to remove this?

Reply:
`feather.read_feather()` doesn't accept this keyword.